.github/workflows/ci.yaml

```diff
@@ -170,6 +169,18 @@ jobs:
             exit 1
           }
 
+      - name: Prepare Sentry package for release
+        run: |
+          New-Item "package-release" -ItemType Directory
+          Copy-Item "package-dev/*" -Destination "package-release/" -Exclude "Tests", "Tests.meta", "package.json" -Recurse
```

Comment:
`*.asmdf`, `*.asmdf.meta`  files from `Runtime` and `Editor` should not be copied, as they contain `dev` endings in file names and config itself.

```json
{
	"name": "io.sentry.unity.dev.editor"
}
```

Reply:
Other `.meta` files are fine?